README.md

```diff
@@ -36,17 +36,26 @@ To learn about Cosmos SDK, please refer [Cosmos SDK Docs](https://github.com/cos
 ```
 make build
 make install
-simd version    # you can see the version!
+
+# you can see the version!
+simd version
 ```
 
 **Configure**
 ```
-sh init_node.sh sim
+sh init_node.sh sim {N(number of nodes), default=1}

```

Comment:
I think you should replace "sh" into "zsh". Some lines of the script contain zsh specific features which cause errors on dash.